proto/s2s.go

```diff
@@ -133,6 +133,14 @@ func (js JobStatusByRuntime) Len() int           { return len(js) }
 func (js JobStatusByRuntime) Less(i, j int) bool { return js[i].Runtime > js[j].Runtime }
 func (js JobStatusByRuntime) Swap(i, j int)      { js[i], js[j] = js[j], js[i] }
 
+type JobLogs []JobLog
+
+func (jls JobLogs) Len() int { return len(jls) }
+func (jls JobLogs) Less(i, j int) bool {
+	return jls[i].RequestId+jls[i].JobId < jls[j].RequestId+jls[j].JobId
+}
+func (jls JobLogs) Swap(i, j int) { jls[i], jls[j] = jls[j], jls[i] }
```

Comment:
I wonder if we're doing this "wrong". The Golang https://golang.org/pkg/sort/ pkg examples have type names like "FooByTs". So I wonder if it's more idiomatic to name these types as such (i.e. "...By...") to indicate that they're not just new types but specifically for sorting. What do you all think?